float32 a
float32 b
float32 c
---
float32 a_to
float32 b_to
float32 c_to

